tests/python/relax/frontend/test_onnx_frontend.py

```diff
@@ -0,0 +1,1378 @@
+    graph = helper.make_graph(
+        [tanh_node],
+        "tanh_test",
+        inputs=[
+            helper.make_tensor_value_info("x", TensorProto.FLOAT, shape),
+        ],
+        outputs=[helper.make_tensor_value_info("y", TensorProto.FLOAT, shape)],
+    )
+
+    model = helper.make_model(graph, producer_name="tanh_test")
+    check_correctness(model)
+
+
+def test_sqrt():
+    sqrt_node = helper.make_node("Sqrt", ["x"], ["y"])
+    shape = [32, 32]
+    graph = helper.make_graph(
+        [sqrt_node],
+        "sqrt_test",
+        inputs=[
+            helper.make_tensor_value_info("x", TensorProto.FLOAT, shape),
+        ],
+        outputs=[helper.make_tensor_value_info("y", TensorProto.FLOAT, shape)],
+    )
+
+    model = helper.make_model(graph, producer_name="sqrt_test")
+    check_correctness(model)
+
+
+def test_relu():
+    relu_node = helper.make_node("Relu", ["x"], ["y"])
+    shape = [32, 32]
+    graph = helper.make_graph(
+        [relu_node],
+        "relu_test",
+        inputs=[
+            helper.make_tensor_value_info("x", TensorProto.FLOAT, shape),
+        ],
+        outputs=[helper.make_tensor_value_info("y", TensorProto.FLOAT, shape)],
+    )
+
+    model = helper.make_model(graph, producer_name="relu_test")
+    check_correctness(model)
+
+
+def test_conv():
+    conv_node = helper.make_node("Conv", ["x", "w", "b"], ["y"])
+    nchw_shape = [3, 12, 32, 32]
+    graph = helper.make_graph(
+        [conv_node],
+        "conv_test",
+        inputs=[
+            helper.make_tensor_value_info("x", TensorProto.FLOAT, nchw_shape),
+            helper.make_tensor_value_info("w", TensorProto.FLOAT, [4, 12, 3, 3]),
+            helper.make_tensor_value_info("b", TensorProto.FLOAT, [4]),
+        ],
+        outputs=[helper.make_tensor_value_info("y", TensorProto.FLOAT, [3, 4, 30, 30])],
+    )
+
+    model = helper.make_model(graph, producer_name="conv_test")
+    check_correctness(model)
+
+
+def test_pow():
+    pow_node = helper.make_node("Pow", ["x", "y"], ["z"])
+    shape = [32, 32]
+    graph = helper.make_graph(
+        [pow_node],
+        "pow_test",
+        inputs=[
+            helper.make_tensor_value_info("x", TensorProto.FLOAT, shape),
+            helper.make_tensor_value_info("y", TensorProto.FLOAT, shape),
+        ],
+        outputs=[helper.make_tensor_value_info("z", TensorProto.FLOAT, shape)],
+    )
+
+    model = helper.make_model(graph, producer_name="pow_test")
+    check_correctness(model)
+
+
+def test_erf():
+    erf_node = helper.make_node("Erf", ["x"], ["y"])
+    shape = [32, 32]
+    graph = helper.make_graph(
+        [erf_node],
+        "erf_test",
+        inputs=[
+            helper.make_tensor_value_info("x", TensorProto.FLOAT, shape),
+        ],
+        outputs=[helper.make_tensor_value_info("y", TensorProto.FLOAT, shape)],
+    )
+
+    model = helper.make_model(graph, producer_name="erf_test")
+    check_correctness(model)
+
+
+def test_cumsum():
```

Comment:
There are multiple cases included in the code generator for this op, like including "reverse" and "exclusive" attributes. These should be tested as well.